src/slices/checkout-slice.ts

```diff
@@ -31,12 +38,57 @@ export const checkoutSlice = createSlice({
     },
 });
 
-export const { setCheckoutView, toggleIsCollapsedCheckoutPanelOpen, toggleIsLoading } = checkoutSlice.actions;
+export const { setCheckoutView, setSelectedCheckoutValueId, toggleIsCollapsedCheckoutPanelOpen, toggleIsLoading } =
+    checkoutSlice.actions;
 
 export const selectLoading = ({ checkout: { loading } }: RootState): boolean => loading;
 
 export const selectCheckoutView = ({ checkout: { view } }: RootState): ViewEnum => view;
 
+export const selectSelectedCheckoutValueId = ({ checkout: { selectedCheckoutValueId } }: RootState): string =>
```

Comment:
Hi Dave, as you can see here I like to have detailed selectors in the redux slice to avoid placing this logic in the React components. I want to keep the React components as dumb as possible. Also, I feel like it's easier to write tests around Redux selectors because they are simply short functions. Is this pattern acceptable for the project? The alternative would be to import the `PrizeoutOfferValueOptions` into a React component and destructure/clean the data there..